src/lib/autocomplete/autocomplete.ts

```diff
@@ -9,16 +9,24 @@ import {
   ViewChild,
   ViewEncapsulation,
   ChangeDetectorRef,
+  Output,
+  EventEmitter,
 } from '@angular/core';
-import {MdOption} from '../core';
+import {MdOption} from '../core/option/option';
 import {ActiveDescendantKeyManager} from '../core/a11y/activedescendant-key-manager';
 
+
 /**
  * Autocomplete IDs need to be unique across components, so this counter exists outside of
  * the component definition.
  */
 let _uniqueAutocompleteIdCounter = 0;
 
+/** Event object that is emitted when an autocomplete option is selected */
+export class MdAutocompleteSelect {
```

Comment:
How about `MdAutocompleteSelectEvent` ? Without "Event", it almost sounds like a hybrid md-autocomplete / md-select directive.